packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -1111,12 +1110,15 @@ describe('postgresql views re-introspection warnings', () => {
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                // 
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                // These views were enriched with \`@@map\` information taken from the previous Prisma schema.
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                // - View "Renamedif"
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                // 
-                                                                                                                                                                                                                                                                                                                                                                        `)
+        // *** WARNING ***
+        // 
+        // These views were enriched with \`@@map\` information taken from the previous Prisma schema:
+        //   - Renamedif
+        // 
+        // These views were enriched with \`@@map\` information taken from the previous Prisma schema:
+        //   - Renamedif
```

Comment:
This seems duplicated

Reply:
Ok, this seems like a bug in https://github.com/prisma/prisma-engines/pull/3906.

In Rust, there's a test `reserved_name_docs_are_only_added_once` that only uses `model`s, in which the `These models were enriched with @@map` string appears only once. 

In TypeScript, on the other hand, we have an equivalent test using `view`s. I'll add this new test to Rust and see how to fix the duplication bug.

Reply:
I also found a similar duplication in the Rust PR [here](https://github.com/prisma/prisma-engines/pull/3906/files/f7209cce364692930db56e825b1c3c20dc04d678#diff-c0e1a4545b7ad78a4dd6787b470e104ac459645f3e16fe26980e673c966408f5)